python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -10261,3 +10261,15 @@ def __getitem__(self, key):
 
     with pytest.raises(TypeError):
         cudf.DataFrame({"a": A()})
+
+
+def test_dataframe_constructor_from_namedtuple():
+    Point1 = namedtuple("Point1", ["a", "b", "c"])
+    Point2 = namedtuple("Point1", ["x", "y"])
+
+    data = [Point1(1, 2, 3), Point2(4, 5)]
```

Comment:
Can you test with the points the other way round in the list? I think it will not work.

Reply:
I can add a test that this fails like pandas does when the points are the other way around

```
In [1]: import numpy as np, pandas as pd

In [2]: from collections import namedtuple

In [3]:     Point1 = namedtuple("Point1", ["a", "b", "c"])
   ...:     Point2 = namedtuple("Point1", ["x", "y"])

In [4]: pd.DataFrame([Point2(4, 5), Point1(1, 2, 3)])
ValueError: 2 columns passed, passed data had 3 columns
```

Reply:
Thanks.